src/ontology/uberon-edit.obo

```diff
@@ -224330,6 +224330,21 @@ relationship: composed_primarily_of CL:0000057 ! fibroblast
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-10-03T14:38:17Z xsd:dateTime
 
+[Term]
+id: UBERON:8480035
+name: cervical transformation zone epithelium
+def: "The area of epithelium that lies between the native and unaffected columnar epithelium of the endocervical canal and the native squamous epithelium deriving from the vaginal and ectocervical squamous epithelium." [https://www.ncbi.nlm.nih.gov/books/NBK568392/]
+comment: Before puberty and after menopause the transformation zone resides in the cervical canal. During the reproductive age of the woman it is located just outside the external os. Squamous cervical cancer originates in the transition zone. Glandular cervical cancer originates in either the transformation zone or the glandular epithelium above the transformation zone. The cervical transformation zone epithelium is an area where squamous metaplasia occurs. {xref="PMID:28639968", xref="PMID:10728686"}
+synonym: "cervical transformation zone" BROAD [PMID:29944714]
+synonym: "cervix transition zone" BROAD [PMID:28932806]
+synonym: "transformation (transitional) zone" BROAD [https://myhealth.alberta.ca/Health/Pages/conditions.aspx?hwid=stt11635]
+synonym: "transformation zone of the cervix" BROAD [https://myhealth.alberta.ca/Health/Pages/conditions.aspx?hwid=stt11635]
+is_a: UBERON:0000483 ! epithelium
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0000002 ! uterine cervix
+property_value: curator_notes "Some sources describe this zone as containing metaplastic squamous cells" xsd:string {xref="PMID:29944714", xref="PMID:30337615"}
```

Comment:
This may be redundant since it is now mentioned in the comment.

Reply:
@bvarner-ebi yes, you are right, I will change